src/include/sof/lib/alloc.h

```diff
@@ -255,4 +255,10 @@ void free_heap(enum mem_zone zone);
 void heap_trace_all(int force);
 void heap_trace(struct mm_heap *heap, int size);
 
+/* retrieve memory map pointer */
+static inline struct mm *memmap_get(void)
+{
+	return sof_get()->memory_map;
```

Comment:
I feel this commit could be squashed with the previous? I'm not strongly in favor of keeping vs squashing so it's your choice.

Reply:
@paulstelian97 I am fan of limiting the number of commits, but there were many requests in the past to keep some changes in separate commits, so I'm trying to match those requests.